ProjectSettings/TagManager.asset

```diff
@@ -3,7 +3,9 @@
 --- !u!78 &1
 TagManager:
   serializedVersion: 2
-  tags: []
+  tags:
+  - Target
+  - Cruz
```

Comment:
Could you explain why the tag “cruz” is in Spanish, it would be better if all the content of the code was in English.